schemas/logs/aws/tests/waf_web_acl_tests.yml

```diff
@@ -0,0 +1,621 @@
+name: waf_web_acl_no_match
+input: |
+  {
+    "timestamp": 1610452695501,
+    "formatVersion": 1,
+    "webaclId": "arn:aws:wafv2:us-west-2:123456789EXAMPLE:regional/webacl/panther_web_dev/f96fe2f4-a2f4-4c06-a4bc-afd495bd675f",
+    "terminatingRuleId": "Default_Action",
+    "terminatingRuleType": "REGULAR",
+    "action": "ALLOW",
+    "terminatingRuleMatchDetails": [],
+    "httpSourceName": "ALB",
+    "httpSourceId": "123456789EXAMPLE-app/web/768dd9170f08bb0d",
+    "ruleGroupList": [],
+    "rateBasedRuleList": [],
+    "nonTerminatingMatchingRules": [],
+    "httpRequest": {
+      "clientIp": "10.0.0.1",
+      "country": "GR",
+      "headers": [
+        {
+          "name": "Host",
+          "value": "web-123456789.us-west-2.elb.amazonaws.com"
+        },
+        {
+          "name": "user-agent",
+          "value": "curl/7.64.1"
+        },
+        {
+          "name": "accept",
+          "value": "*/*"
+        }
+      ],
+      "uri": "/",
+      "args": "o=11 AND 1=1",
+      "httpVersion": "HTTP/2.0",
+      "httpMethod": "HEAD",
+      "requestId": "1-5ffd8ed7-1c9649c448649a15241ac25a"
+    }
+  }
+result: |
+  {
+      "timestamp": 1610452695501,
+      "formatVersion": 1,
+      "webaclId": "arn:aws:wafv2:us-west-2:123456789EXAMPLE:regional/webacl/panther_web_dev/f96fe2f4-a2f4-4c06-a4bc-afd495bd675f",
+      "terminatingRuleId": "Default_Action",
+      "terminatingRuleType": "REGULAR",
+      "action": "ALLOW",
+      "httpSourceName": "ALB",
+      "httpSourceId": "123456789EXAMPLE-app/web/768dd9170f08bb0d",
+      "httpRequest": {
+        "clientIp": "10.0.0.1",
+        "country": "GR",
+        "headers": [
+          {
+            "name": "Host",
+            "value": "web-123456789.us-west-2.elb.amazonaws.com"
+          },
+          {
+            "name": "user-agent",
+            "value": "curl/7.64.1"
+          },
+          {
+            "name": "accept",
+            "value": "*/*"
+          }
+        ],
+        "uri": "/",
+        "args": "o=11 AND 1=1",
+        "httpVersion": "HTTP/2.0",
+        "httpMethod": "HEAD",
+        "requestId": "1-5ffd8ed7-1c9649c448649a15241ac25a"
+      },
+      "p_event_time": "2021-01-12T11:58:15.501Z",
+      "p_any_ip_addresses": ["10.0.0.1"],
+      "p_any_trace_ids": ["1-5ffd8ed7-1c9649c448649a15241ac25a"],
+      "p_log_type": "AWS.WAFWebACL"
```

Comment:
> p_any_domain_names for web-123456789.us-west-2.elb.amazonaws.com

Note that it would require special logic to apply this extraction only for [the `Host` header](https://github.com/panther-labs/panther/blob/315721e897cf42bb6eab188a51fb7c0a28023215/internal/log_analysis/log_processor/parsers/awslogs/waf_web_acl.go#L98). I am quite new at the codebase, but my understanding is that we can only define a scanner on the field level, and not conditionally according to another field value.

> p_any_aws_arns for arn:aws:wafv2:us-west-2:123456789EXAMPLE:regional/webacl/panther_web_dev/f96fe2f4-a2f4-4c06-a4bc-afd495bd675f

I see this only for specific log parsers (CloudTrail and VPC), so I guess it also requires some custom logic. I believe the same applies to `p_any_aws_account_ids`.

These (and other custom Panther field notes) seem like good observations though, would you mind summarizing them in an issue at https://github.com/panther-labs/panther instead? Thanks!